### PR TITLE
wxmaxima 22.09.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Porkepix** [comment]
Fails to find `wxwidgets` although it's correctly set as a dependency:

```
  -- Looking for wxWidgets libraries in: 
  CMake Error at /home/linuxbrew/.linuxbrew/Cellar/cmake/3.24.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
    Could NOT find wxWidgets (missing: wxWidgets_LIBRARIES) (Required is at
    least version "3")
  Call Stack (most recent call first):
    /home/linuxbrew/.linuxbrew/Cellar/cmake/3.24.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
    /home/linuxbrew/.linuxbrew/Cellar/cmake/3.24.1/share/cmake/Modules/FindwxWidgets.cmake:977 (find_package_handle_standard_args)
    CMakeLists.txt:137 (find_package)
```

**Comment by carlocab** [comment]
Only Linux is failing.

**Comment by cho-m** [comment]
Could be due to `wxmaxima` now requiring webview component. This is in macOS `wxwidgets` due to WebKit framework, but I don't think we currently have it in Linux bottle.

File: Formula/wxwidgets.rb [hunk 1]
```diff
@@ -30,6 +30,7 @@ class Wxwidgets < Formula
     depends_on "gtk+3"
     depends_on "libsm"
     depends_on "mesa-glu"
+    depends_on "webkitgtk"
```
Comment (carlocab) [inline Formula/wxwidgets.rb:33]: `wxwidgets` doesn't seem to be linking with `webkitgtk`, despite `--enable-webviewwebkit` being passed to `configure`.

Comment (carlocab) [reply Formula/wxwidgets.rb:33]: Would be nice to have proper logs here. Linux is really uncooperative sometimes...

Comment (carlocab) [reply Formula/wxwidgets.rb:33]: Ah, I think I see the problem. `configure` looks for `webkit2gtk-4.0`, but we ship `webkit2gtk-4.1`.

**Comment by carlocab** [comment]
```
  ./src/gtk/webview_webkit2.cpp:169:18: error: ‘SOUP_STATUS_CANCELLED’ was not declared in this scope
               case SOUP_STATUS_CANCELLED:
                    ^
```

**Comment by carlocab** [comment]
Ok, I don't know how to fix this. No idea why it's missing things that should be defined by `libsoup`.

In any case, `wxmaxima` has had 9 installs on Linux in the past year (vs ~6000 on macOS), so I don't think the Linux build failure should block this.

**Review by carlocab — DISMISSED** [review]

File: Formula/wxmaxima.rb [hunk 2]
```diff
@@ -13,7 +12,6 @@ class Wxmaxima < Formula
     sha256 monterey:       "2e9c6669e599f27521441335baf9c2f0080b6c046e6bb25b04938175ef600f8d"
     sha256 big_sur:        "7853c4c1e1edb1781cfe06f62b4bfb99e2b75c19452393594c7834ed97046fba"
     sha256 catalina:       "c4849f6ed0a27363d6b3d3ecb8303887ad8d8e2e7841483cbae8bee3ecae2d4b"
-    sha256 x86_64_linux:   "46115cdf5170522d9ec330253e6ad97c98b10dc75c0b4b3b8fb4978a6c155090"
```
Comment (carlocab) [inline Formula/wxmaxima.rb:16]: Removed the Linux bottle to make sure CI passes.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**